pkg/operator/ceph/config/datapath_test.go

```diff
@@ -24,44 +24,54 @@ import (
 
 func TestNewStatefulDaemonDataPathMap(t *testing.T) {
 	// mon
-	d := NewStatefulDaemonDataPathMap("/var/lib/rook", "/mon-a/data", MonType, "a")
+	d := NewStatefulDaemonDataPathMap("/var/lib/rook", "/mon-a/data", MonType, "a", "rook-ceph")
 	assert.Equal(t, &DataPathMap{
 		PersistData:      true,
 		HostDataDir:      "/var/lib/rook/mon-a/data",
 		ContainerDataDir: "/var/lib/ceph/mon/ceph-a",
+		ContainerLogDir:  "/var/log/ceph",
+		HostLogDir:       "/var/lib/rook/log/rook-ceph",
 	}, d)
 
 	// osd
-	d = NewStatefulDaemonDataPathMap("/var/lib/rook/", "osd0/", OsdType, "0")
+	d = NewStatefulDaemonDataPathMap("/var/lib/rook/", "osd0/", OsdType, "0", "rook-ceph")
 	assert.Equal(t, &DataPathMap{
 		PersistData:      true,
 		HostDataDir:      "/var/lib/rook/osd0",
 		ContainerDataDir: "/var/lib/ceph/osd/ceph-0",
+		ContainerLogDir:  "/var/log/ceph",
+		HostLogDir:       "/var/lib/rook/log/rook-ceph",
 	}, d)
 }
 
 func TestNewStatelessDaemonDataPathMap(t *testing.T) {
 	// mgr
-	d := NewStatelessDaemonDataPathMap(MgrType, "a")
+	d := NewStatelessDaemonDataPathMap(MgrType, "a", "rook-ceph", "/var/lib/rook/")
```

Comment:
Could you change one of these `"/var/lib/rook/"` strings to be `"/var/lib/rook"` without the trailing slash? I've encountered the issue in Rook before that it will sometimes treat the same path as a different path depending on whether there is a trailing slash or not. I don't anticipate that actually being an issue here, but it could catch something in the future.